engine/consensus/dkg/reactor_engine.go

```diff
@@ -273,20 +364,22 @@ func (e *ReactorEngine) registerPhaseTransition(view uint64, fromState dkgmodule
 func (e *ReactorEngine) end(epochCounter uint64) func() error {
 	return func() error {
 		err := e.controller.End()
-		if err != nil {
-			if crypto.IsDKGFailureError(err) {
-				e.log.Warn().Msgf("node %s with index %d failed DKG locally: %s", e.me.NodeID(), e.controller.GetIndex(), err.Error())
-			} else {
-				return err
+		if crypto.IsDKGFailureError(err) {
```

Comment:
Just making sure that `crypto.IsDKGFailureError(nil)` returns `false` ?

Reply:
👍  it does -> https://go.dev/play/p/6nHREXi5R0O